include/bluetooth/services/fast_pair.h

```diff
@@ -99,6 +99,18 @@ struct bt_fast_pair_battery {
 	uint8_t level;
 };
 
+/** Fast Pair information callback descriptor. */
+struct bt_fast_pair_info_cb {
+	/** Notify that the Account Key list has been updated.
+	 *
+	 * This information can be used to update the Fast Pair advertising data
+	 * in the not discoverable mode.
+	 *
+	 *  @param conn Connection object that changed the Account Key list.
+	 */
+	void (*account_key_list_changed)(struct bt_conn *conn);
```

Comment:
You could also inform about the context from which the callback is called. Zephyr documentation states that:
`Attribute read and write callbacks are called directly from RX Thread thus it is not recommended to block for long periods of time in them.`

Reply:
> `account_key_written`/`account_key_write`? The callback is not called when keys are removed (e.g. during factory reset), so current naming can be slightly confusing. Since factory reset is triggered by the application, I would not call information callback in that case - application is aware of the fact that the keys are removed during factory reset.

Discussed offline. We decided to keep the name and call the callback also during the factory reset operation.